localenv/README.md

```diff
@@ -109,6 +109,35 @@ The secondary Happy Life Bank docker compose file (`./happy-life-bank/docker-com
 data stores created by the primary Rafiki instance so it can't be run by itself.
 The `pnpm localenv:compose up` command starts both the primary instance and the secondary.
 
+#### Autopeering
+
+If you want to start the local env and peer it automatically to rafiki.money, you can run the following commands:
+
+```
+pnpm localenv:autopeer:start
```

Comment:
```suggestion
pnpm localenv:autopeer:start -d
```

Reply:
I guess the logs are required for finding out the wallet address, though 🤔. Can we log that some other way?

Reply:
added it in the command and aditional instructions how to find the payment pointer url from docker logs (it will be much easier to find it in ase logs)